src/test/kotlin/com/terraformation/backend/db/GeometryDeserializerTest.kt

```diff
@@ -1,165 +0,0 @@
-package com.terraformation.backend.db
```

Comment:
It turned out nearly all these cases were already covered by GeometrySerializerTest, which was added after this class. I added the couple of missing cases to that class and there's no more need for this one.